### DISCUSSION TITLE
文件夹里的文件都没了？是那个pdb文件吗？

### DISCUSSION THREAD

**Discussion by wlkzs666** [c0re100/qBittorrent-Enhanced-Edition#282] [General]
想加个搜索插件，但是找不到那个nova3文件夹了，只有个一百多兆的pdb文件

**Comment by wlkzs666** [comment]
安装选项里有一个一并安装调试符号文件，去掉那个勾pdb就没了，但是还是没有插件的安装文件夹，想安一个jackett插件，还有别的方法吗

**Comment by c0re100** [comment]
插件是從QB裡安裝的...

**Comment by wlkzs666** [comment]
> 插件是从QB里安装的...

嗯，我是想安jackett插件，要替换那个json里的api，就是找不着文件夹

**Comment by wlkzs666** [comment]
> 插件是從QB裡安裝的...

我看好像也有插件格式的，不过连接好像挂了，下不了，只能改api了

**Comment by wlkzs666** [comment]
安了个基础版的试了一下，也是没有搜索插件的安装目录，只有一个一百多兆的pdb，是不是要把这个pdb文件打开呀

**Comment by c0re100** [comment]
https://github.com/qbittorrent/search-plugins/wiki/Install-search-plugins

**Reply by wlkzs666** [reply]
https://github.com/qbittorrent/search-plugins/wiki/How-to-configure-Jackett-plugin
jackett的py文件下不了，不能本地安装，所以我才要找文件夹的，qbittorrent提供的插件里也没有

**Comment by wlkzs666** [comment]
哦，路径应该是c\users\username\appdata\local\qbittorrent,不是在那个安装目录里
